startup.sh

```diff
@@ -12,10 +12,10 @@
 
 SUBSONIC_HOME=/var/subsonic
 SUBSONIC_HOST=0.0.0.0
-SUBSONIC_PORT=4040
-SUBSONIC_HTTPS_PORT=0
-SUBSONIC_CONTEXT_PATH=/
-SUBSONIC_MAX_MEMORY=200
+SUBSONIC_PORT=${HTTP_PORT}
+SUBSONIC_HTTPS_PORT=${HTTPS_PORT}
```

Comment:
Now that we define these three environment variables based on other variables, we have two choices:

1. Define default HTTPS_PORT, CONTEXT_PATH, and MAX_MEM with an `ENV` in the Dockerfile, so we don't have to type them when using docker run.

2. Define defaults here, like:

```
SUBSONIC_HTTPS_PORT=${HTTPS_PORT:-0}
SUBSONIC_CONTEXT_PATH=${CONTEXT_PATH:-/}
SUBSONIC_MAX_MEMORY=${MAX_MEM:-200}
```

This would use shell constructs to set defaults if those variables are not set.

I'm OK with either. Just pick one of the two methods above (and if you have a few minutes to spare, run a quick test to make sure it works as expected). :)